eras/conway/impl/src/Cardano/Ledger/Conway/Governance/DRepPulser.hs

```diff
@@ -298,20 +316,31 @@ class
   runConwayRatify globals ratifyEnv ratifyState (RatifySignal ratifySig) =
     let ratifyResult =
           runReader
-            ( applySTS @(ConwayRATIFY era) (TRC (ratifyEnv, ratifyState, RatifySignal $ reorderActions ratifySig))
+            ( applySTS @(ConwayRATIFY era) $
+                TRC (ratifyEnv, ratifyState, RatifySignal $ reorderActions ratifySig)
             )
             globals
      in case ratifyResult of
           Left ps ->
-            error (unlines ("Impossible: RATIFY rule never fails, but it did:" : map show (toList ps)))
+            error $
+              unlines $
+                "Impossible: RATIFY rule never fails, but it did:"
+                  : map show (toList ps)
           Right ratifyState' -> ratifyState'
 
 finishDRepPulser :: DRepPulsingState era -> (PulsingSnapshot era, RatifyState era)
 finishDRepPulser (DRComplete snap ratifyState) = (snap, ratifyState)
 finishDRepPulser (DRPulsing (DRepPulser {..})) =
   (PulsingSnapshot dpProposals finalDRepDistr dpDRepState, ratifyState')
   where
-    !finalDRepDistr = computeDRepDistr dpStakeDistr dpDRepState dpDRepDistr $ Map.drop dpIndex $ umElems dpUMap
+    !finalDRepDistr =
+      computeDRepDistr
+        dpStakeDistr
+        dpDRepState
+        dpProposals
+        dpDRepDistr
+        $ Map.drop dpIndex
+        $ umElems dpUMap
```

Comment:
A local binding will make it possible to avoid this column style
```suggestion
    !leftOver = Map.drop dpIndex $ umElems dpUMap
    !finalDRepDistr =
      computeDRepDistr dpStakeDistr dpDRepState dpProposals dpDRepDistr leftOver
```